cmd/sup/main.go

```diff
@@ -99,6 +100,12 @@ func parseArgs(conf *sup.Supfile) (*sup.Network, []*sup.Command, error) {
 	// Add default env variable with current network
 	network.Env["SUP_NETWORK"] = args[0]
 
+	// Add default nonce
+	network.Env["SUP_NONCE"] = time.Now().Format("2006-01-02T15:04:05")
```

Comment:
Not sure I like the Unix timestamp - it takes too long to figure out when the deploy was done. 

The Format used was done to reflect https://golang.org/pkg/time/#RFC3339 without timezone part, in order to keep it short. But since we're making up our own format anyway I see no reason we shouldn't go with your version. 

Completely agree with UTC - will fix.